src/test/kotlin/io/github/mojira/arisa/modules/HelpersTest.kt

```diff
@@ -0,0 +1,305 @@
+
+        moduleEither.shouldBeRight()
+        (moduleEither as Either.Right).b.shouldBe(ModuleResponse)
+    }
+
+    "assertNotEquals when given equal arguments should return OperationNotNeededModuleResponse in Either" {
+        val moduleEither = assertNotEquals(1337, 1337)
+
+        moduleEither.shouldBeLeft()
+        (moduleEither as Either.Left).a.shouldBe(OperationNotNeededModuleResponse)
+    }
+
+    "assertTrue when given true should return ModuleResponse in Either" {
+        val moduleEither = assertTrue(true)
+
+        moduleEither.shouldBeRight()
+        (moduleEither as Either.Right).b.shouldBe(ModuleResponse)
+    }
+
+    "assertTrue when given false should return OperationNotNeededModuleResponse in Either" {
+        val moduleEither = assertTrue(false)
+
+        moduleEither.shouldBeLeft()
+        (moduleEither as Either.Left).a.shouldBe(OperationNotNeededModuleResponse)
+    }
+
+    "assertFalse when given false should return ModuleResponse in Either" {
+        val moduleEither = assertFalse(false)
+
+        moduleEither.shouldBeRight()
+        (moduleEither as Either.Right).b.shouldBe(ModuleResponse)
+    }
+
+    "assertFalse when given true should return OperationNotNeededModuleResponse in Either" {
+        val moduleEither = assertFalse(true)
+
+        moduleEither.shouldBeLeft()
+        (moduleEither as Either.Left).a.shouldBe(OperationNotNeededModuleResponse)
+    }
+
+    "assertGreaterThan when given 2 arguments first of which is greater than the second should return ModuleResponse in Either" {
+        val moduleEither = assertGreaterThan(2, 1)
+
+        moduleEither.shouldBeRight()
+        (moduleEither as Either.Right).b.shouldBe(ModuleResponse)
+    }
+
+    "assertGreaterThan when given 2 arguments first of which is smaller than the second should return OperationNotNeededModuleResponse in Either" {
+        val moduleEither = assertGreaterThan(1, 2)
+
+        moduleEither.shouldBeLeft()
+        (moduleEither as Either.Left).a.shouldBe(OperationNotNeededModuleResponse)
+    }
+
+    "assertGreaterThan when given 2 arguments that are equal to each other should return OperationNotNeededModuleResponse in Either" {
+        val moduleEither = assertGreaterThan(1, 1)
+
+        moduleEither.shouldBeLeft()
+        (moduleEither as Either.Left).a.shouldBe(OperationNotNeededModuleResponse)
+    }
+
+    "assertAfter when given 2 times first of which is after the second should return ModuleResponse in Either" {
+        val moduleEither = assertAfter(NOW, A_SECOND_AGO)
+
+        moduleEither.shouldBeRight()
+        (moduleEither as Either.Right).b.shouldBe(ModuleResponse)
+    }
+
+    "assertAfter when given 2 times first of which is before the second should return OperationNotNeededModuleResponse in Either" {
+        val moduleEither = assertAfter(A_SECOND_AGO, NOW)
+
+        moduleEither.shouldBeLeft()
+        (moduleEither as Either.Left).a.shouldBe(OperationNotNeededModuleResponse)
```

Comment:
moduleEither shouldBeLeft OperationNotNeededModuleResponse

same with Right :P, that replaces the .shouldBeLeft() and the casting for shouldBe

Reply:
ah ok

Reply:
done in https://github.com/mojira/arisa-kt/pull/433/commits/7c5bddd90ca644a8f09e26c293a336d735ceb72f